weblate/examples/apache.conf

```diff
@@ -3,38 +3,53 @@
 #
 # This example assumes Weblate is installed in virtualenv in /home/weblate/weblate-env
 # and DATA_DIR is set to /home/weblate/data, please adjust paths to match your setup.
-#
+# Replace <your_domain> with your domain
+
 <VirtualHost *:80>
-    ServerAdmin admin@weblate.example.org
-    ServerName weblate.example.org
+    ServerName <your_domain>
+
+    ErrorLog ${APACHE_LOG_DIR}/error.log
+    CustomLog ${APACHE_LOG_DIR}/access.log combined
+
+    RewriteEngine on
+    RewriteRule ^ https://%{HTTP_HOST}%{REQUEST_URI} [R=301,L]
+</VirtualHost>
+
+<VirtualHost *:443>
+    ServerAdmin admin@<your_domain>
+    ServerName <your_domain>
 
     # DATA_DIR/static/favicon.ico
-    Alias /favicon.ico /home/weblate/data/static/favicon.ico
+    Alias /favicon.ico /home/weblate/lib/python3.9/site-packages/data/static/favicon.ico
```

Comment:
The DATA_DIR is better to be placed outside the package directories.

Reply:
It's path where Weblate stores data, it is configured in the settings.

Reply:
Maybe so, I just have to set up the above so that Weblate starts at all...

Reply:
Okay, let's guide users towards configuring this in https://github.com/WeblateOrg/weblate/pull/7858